### ISSUE TITLE
Fix `declaration-property-value-no-unknown` false positives for `overflow: overlay`

### ISSUE THREAD

**Issue by silverwind** [stylelint/stylelint#6667]
### What steps are needed to reproduce the bug?

https://stylelint.io/demo/#N4Igxg9gJgpiBcIBG0CeACYAdAdu9EAbjAE4BmANhAO7wHEkUCGqA3LgL64gA0IAzqhwAXJgA8E4fv17gIOMgEsA5pOx50WECQCuFGPy111+fFthhmJJsMXyAtAAcSER6WGp7hJhR0x7OBD2OjgA1oHUOEbowrowuPhcOBwgHEA

This is a lesser-known property that is still supported in Blink. [MDN reference](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow#overlay).

### What Stylelint configuration is needed to reproduce the bug?

See demo

### How did you run Stylelint?

demo

### Which version of Stylelint are you using?

15.1.0

### What did you expect to happen?

No error

### What actually happened?

`Unexpected unknown value "overlay" for property "overflow"`

### Does the bug relate to non-standard syntax?

No

### Proposal to fix the bug

Add the property to upstream dependencies. I checked `csstree` and `mdn/data` repos, but wasn't able to find where the definition for valid `overflow` values actually resides. Pointers appreciated.

**Comment by jeddy3** [comment]
@silverwind Thanks for the report and for using the template.

> Add the property to upstream dependencies. I checked csstree and mdn/data repos, but wasn't able to find where the definition for valid overflow values actually resides. Pointers appreciated.

I think these are the relevant files and lines:

- [MDN data](https://github.com/mdn/data/blob/678604eeb7cad0c1e5d637fa6918922be1b24be3/css/properties.json#L7325-L7356)
- [CSSTree patch](https://github.com/csstree/csstree/blob/612cc5f2922b2304869497d165a0cc65257f7a8b/data/patch.json#L551-L554)

As it's a non-standard value, it probably belongs in the patch file.

Please consider contributing upstream if you have time.

You can also [extend the syntax](https://stylelint.io/user-guide/rules/declaration-property-value-no-unknown/#propertiessyntax--property-syntax-) in your config.

To extend the `overflow` property ([Demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcICGACYKIDcYCcBmANhAO7wbY4FICeA3CgL4gA0IAztQHYAuSAHgnBs2LcBE54AlgHNBwADqcUKeSBwBXAjDaqyCpcpUhYYKjiTdJ4gLQAHHBFu5u1a5iQF1Ma5wjX1nADWvsScuigA2twaMMxoRvaOzpLaAMpcvHzh+oaGqli4hCThqgA+5LhU1KqKuQwMALq1jIpMDEA)):

```json
{
  "rules": {
    "declaration-property-value-no-unknown": [true, { "propertiesSyntax": {
      "overflow": "| overlay"
    }}]
  }
}
```

Or the underlying type ([Demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcICGACYKIDcYCcBmANhAO7wbY4FICeA3CgL4gA0IAztQHYAuSAHgnBs2LcBE54AlgHNBwADqcUKeSBwBXAjDaqyCpcpUhYYKjiTdJ4gLQAHHBFu5u1a5iQF1Ma5wjX1nADWvsScuigA2twaMMxoRi5ObADKXLx84fqGhqo+Nmy8nFBIOFDWWLiEJOGqAD7kuFTUqorZDAwAuq2MikwMQA)):

```json
{
  "rules": {
    "declaration-property-value-no-unknown": [true, { "typesSyntax": {
      "-non-standard-overflow": "| overlay"
    }}]
  }
}
```

**Comment by silverwind** [comment]
Thanks, yes I will contribute it in csstree.

**Comment by silverwind** [comment]
https://github.com/csstree/csstree/pull/248

**Comment by silverwind** [comment]
csstree merged it, should be in next release.

**Comment by silverwind** [comment]
Still waiting on next `css-tree` package release.
